README.md

```diff
@@ -59,15 +58,16 @@ where the environment variable `PATH_TO_CHROME_KEY` points to the extension's ke
 
 It should also be possible to just load a zipped version of the `build` directory generated by `npm run pack` in any browser supporting WebExtensions, but tools might do other things too.
 
-## Run Extension in Safari
-To run the extension in Safari, you need to enable the Develop menu in Safari. To do this, go to Safari > Preferences > Advanced and check the box next to "Show Develop menu in menu bar". Then, in the Develop menu, select "Allow Unsigned Extensions".
+## EARLY BETA: Safari Extension (desktop and iOS)
+The Chrome extension can be converted into an extension that will run in desktop Safari, or (with a separate conversion) in Safari on iOS. To build and run the extension in Safari:
```

Comment:
Everything below this line seems like it is redundant with what is better described in Safari.md. Can we just point to Safari.md, and only have these instructions in one place?